### PR TITLE
Add ptags

### PR DESCRIPTION
**Pull request by matthewjberger**
ptags is a wrapper around ctags (a faster version of universal-ctags)

### CODE REVIEW

File: bucket/ptags.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.3.0",
+    "description": "A parallel universal-ctags wrapper for git repository",
+    "license": "MIT",
+    "url": "https://github.com/dalance/ptags/releases/download/v0.3.0/ptags-v0.3.0-x86_64-win.zip",
+    "depends": "ctags",
+    "bin": "ptags.exe",
+    "homepage": "https://github.com/dalance/ptags",
```
Comment (Ash258) [inline bucket/ptags.json]: Move before license

File: bucket/ptags.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "hash": "4628fa3ca128a81b802d78ef49c362065eb8c483dde025308447d5214a5221cb",
+    "checkver": {
+        "github": "https://github.com/dalance/ptags"
+    },
```
Comment (Ash258) [inline bucket/ptags.json]:
```suggestion
    "checkver": "github",
```

File: bucket/ptags.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.3.0",
+    "description": "A parallel universal-ctags wrapper for git repository",
+    "license": "MIT",
+    "url": "https://github.com/dalance/ptags/releases/download/v0.3.0/ptags-v0.3.0-x86_64-win.zip",
+    "depends": "ctags",
+    "bin": "ptags.exe",
+    "homepage": "https://github.com/dalance/ptags",
+    "hash": "4628fa3ca128a81b802d78ef49c362065eb8c483dde025308447d5214a5221cb",
```
Comment (Ash258) [inline bucket/ptags.json]: Move hash after URL

File: bucket/ptags.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.3.0",
+    "description": "A parallel universal-ctags wrapper for git repository",
```
Comment (Ash258) [inline bucket/ptags.json]:
```suggestion
    "description": "A parallel universal-ctags wrapper for git repository.",
```

File: bucket/ptags.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "license": "MIT",
+    "url": "https://github.com/dalance/ptags/releases/download/v0.3.0/ptags-v0.3.0-x86_64-win.zip",
+    "depends": "ctags",
```
Comment (Ash258) [inline bucket/ptags.json]: Before URL

File: bucket/ptags.json (hunk 1 above)
Comment (matthewjberger) [reply bucket/ptags.json]: url, depends, bin, and homepage or just homepage?

Comment (Ash258) [reply bucket/ptags.json]: Comment is for L8. What exactly make you feel like it was for all lines above?

**Comment by matthewjberger** [comment]
How does the file look now?

**Comment by matthewjberger** [comment]
How does it look?

**Comment by linsui** [comment]
Does this work for 32bit? If not, please put url and hash into architecture.64bit .

**Comment by matthewjberger** [comment]
The url and hash are now in an 64-bit architecture block

**Pull request merged**